src/redux/categories/categories.js

```diff
@@ -0,0 +1,32 @@
+const CHECK_STATUS = 'bookstore/books/CHECK_STATUS';
+const REMOVE_CATEGORY = 'bookstore/books/REMOVE_CATEGORY';
+
+// add book action mathod
+export const addCategory = (payload) => (
+  {
+    type: CHECK_STATUS,
+    payload,
+
+  });
+
+export const removeBook = (id) => (
+  {
+    type: REMOVE_CATEGORY,
+    id,
+  });
+
+const categoryReducer = (state = [], action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return [
+        ...state,
+        ...action.payload,
+      ];
```

Comment:
- [ ] Kindly notice that in case `action.type` is `CHECK_STATUS` the function returns a string "Under construction".

```javascript
 switch (action.type) {
    case CHECK_STATUS:
      return 'Under construction';
    default:
      return state;
}
```